db/go/util.go

```diff
@@ -48,6 +48,51 @@ func updateWorkspaceTemplateManifest(filename, templateName string) error {
 	return nil
 }
 
+// createWorkflowTemplate will create the workflow template given by {{templateName}} with the contents
+// given by {{filename}}
+// It will do so for all namespaces.
+func createWorkflowTemplate(filename, templateName string, labels map[string]string) error {
+	client, err := getClient()
+	if err != nil {
+		return err
+	}
+	defer client.DB.Close()
+
+	namespaces, err := client.ListOnepanelEnabledNamespaces()
+	if err != nil {
+		return err
+	}
+
+	manifest, err := readDataFile(filename)
+	if err != nil {
+		return err
+	}
+
+	uid, err := uid2.GenerateUID(templateName, 30)
+	if err != nil {
+		return err
+	}
+
+	for _, namespace := range namespaces {
+		workflowTemplate := &v1.WorkflowTemplate{
+			UID:      uid,
+			Name:     templateName,
+			Manifest: manifest,
+			Labels:   labels,
+		}
+
+		err = ReplaceArtifactRepositoryType(client, namespace, workflowTemplate, nil)
```

Comment:
@rushtehrani Let's use `ReplaceRuntimeVariablesInManifest`, ` ReplaceArtifactRepositoryType` does not do `select.nodepool`